tests/test_classifier/test_boundaries.py

```diff
@@ -0,0 +1,337 @@
+# tests.test_boundaries
+# Ensure that the decision boundary visualizations work.
+#
+# Author:   Author:   Nathan Danielsen <nathan.danielsen@gmail.com>
+# Created:  Sun Mar 19 13:01:29 2017 -0400
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_knn.py [] nathan.danielsen@gmail.com $
+"""
+Ensure that the Decision Boundary visualizations work.
+"""
+
+##########################################################################
+# Imports
+##########################################################################
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from collections import OrderedDict
+import numpy as np
+
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
+
+import unittest
+from tests.base import VisualTestCase
+from yellowbrick.classifier import *
+from yellowbrick.exceptions import YellowbrickTypeError
+from yellowbrick.exceptions import YellowbrickValueError
+
+from sklearn import datasets
+from sklearn import neighbors
+from sklearn import naive_bayes
+##########################################################################
+# Data
+##########################################################################
+# yapf: disable
+X = np.array([
+    [2.318, 2.727, 4.260, 7.212, 4.792, ],
+    [2.315, 2.726, 4.295, 7.140, 4.783, ],
+    [2.315, 2.724, 4.260, 7.135, 4.779, ],
+    [2.110, 3.609, 4.330, 7.985, 5.595, ],
+    [2.110, 3.626, 4.330, 8.203, 5.621, ],
+    [2.110, 3.620, 4.470, 8.210, 5.612, ],
+    [2.318, 2.727, 4.260, 7.212, 4.792, ],
+    [2.315, 2.726, 4.295, 7.140, 4.783, ],
+    [2.315, 2.724, 4.260, 7.135, 4.779, ],
+    [2.110, 3.609, 4.330, 7.985, 5.595, ],
+    [2.110, 3.626, 4.330, 8.203, 5.621, ],
+    [2.110, 3.620, 4.470, 8.210, 5.612, ]
+    ])
+
+y = np.array([1, 2, 1, 2, 1, 0, 0, 1, 3, 1, 3, 2])
+
+X_two_cols = X[:, :2]
+# yapf: enable
+##########################################################################
+# Residuals Plots test case
+##########################################################################
+
+
+class DecisionBoundariesVisualizerTest(VisualTestCase):
+    """Testcases for the DecisionBoundariesVisualizers """
+
+    def test_decision_bounardies(self):
+        """Assert no errors occur during KnnDecisionBoundariesVisualizer
```

Comment:
With the -v flag on nosetests set, the first line of the docstring is used in the test description; this isn't a big deal, but just wanted to note that the word "integration" would be chopped off during tests. I think it's pretty readable even so.